balancer/endpointsharding/endpointsharding_test.go

```diff
@@ -102,6 +102,10 @@ type fakePetiole struct {
 	bOpts balancer.BuildOptions
 }
 
+func (fp *fakePetiole) ExitIdle() {
+	fp.Balancer.ExitIdle()
```

Comment:
This isn't necessary, is it?  `Balancer` is embedded so this should happen automatically?